sdk/react/panels/cast/CastDevicesScreen.js

```diff
@@ -0,0 +1,189 @@
+
+    return (
+      <Animated.View
+        style={[styles.fullscreenContainer, animationStyle, {
+          height,
+          width,
+        }]}
+      >
+        <Animated.View
+          style={[animationStyle, {
+            height,
+            width,
+            marginTop: 20,
+          }]}
+        >
+          <ListView
+            style={styles.listViewContainer}
+            dataSource={dataSource}
+            renderRow={(rowData, sectionID, rowID) => this.renderItem(rowData, rowID, castButton, castButtonActive)}
+          />
+        </Animated.View>
+        <Text style={styles.title}>
+          {'Remote playback'}
+        </Text>
+        {dismissButtonRow}
+      </Animated.View>
+    );
+  }
+
+  renderItem(rowData, rowID, castButton, castButtonActive) {
+    const { selectedItem } = this.props;
+    const { config } = this.props;
+    const { iconStyle } = config.castControls;
+    const { selectedID } = this.state;
+
+    const isSelected = selectedID === rowID || rowData === selectedItem;
+    const itemContainerStyle = isSelected ? styles.itemContainerSelected : styles.itemContainer;
+
+    const textColor = isSelected ? iconStyle.active.color : iconStyle.inactive.color;
+    const textStyle = isSelected ? styles.textSelected : styles.text;
+    return (
+      <TouchableHighlight
+        style={{ flex: 1 }}
+        onPress={() => this.onPressButton(rowID)}
+        underlayColor="transparent"
+      >
+        <View style={itemContainerStyle}>
+          <View style={styles.icon}>
+            {isSelected ? castButtonActive : castButton}
+          </View>
+          <Text style={[textStyle, { color: textColor }]}>
+            {rowData}
+          </Text>
+        </View>
+      </TouchableHighlight>
+    );
+  }
+
+  renderCastButton(color) {
+    const { config } = this.props;
+
+    return Utils.renderRectButton(BUTTON_NAMES.CAST,
+      null,
+      config.icons['chromecast-disconnected'].fontString,
+      null, castButtonSize, color,
+      config.icons['chromecast-disconnected'].fontFamilyName);
+  }
+
+  render() {
+    const { config } = this.props;
+    const { opacity } = this.state;
+
+    const dismissButton = Utils.renderRectButton(BUTTON_NAMES.DISMISS,
+      styles.iconDismiss,
+      config.icons.dismiss.fontString,
+      this.onDismissPress, dismissButtonSize,
+      config.castControls.iconStyle.inactive.color,
+      config.icons.dismiss.fontFamilyName);
+
+    const castButton = this.renderCastButton(config.castControls.iconStyle.inactive.color);
+    const castButtonActive = this.renderCastButton(config.castControls.iconStyle.active.color);
+
+    const dismissButtonRow = (
+      <View style={styles.dismissButtonTopRight}>
+        {dismissButton}
+      </View>
+    );
+    const animationStyle = { opacity };
+    return (this.renderCastDevicesScreen(animationStyle, castButton, castButtonActive, dismissButtonRow));
+  }
+}
+
+module.exports = CastDevicesScreen;
```

Comment:
Use `export default class` over `module.exports`

Reply:
We will fix it during the process of lint errors fixing.